package.json

```diff
@@ -8,98 +10,48 @@
   },
   "scripts": {
     "init": "yarn && yarn build",
-    "create-package": "node ./scripts/create-package.js",
+    "create-package": "lg-create",
     "build": "turbo run build tsc",
     "build-storybook": "storybook build",
-    "build:docs": "npx ts-node ./scripts/parse-tsdocs.ts",
-    "lint": "npm-run-all --parallel lint:*",
-    "lint:prettier": "prettier --check \"**/*.{js,ts,tsx,json,md,yml}\"",
-    "lint:eslint": "eslint \"**/*.{js,ts,tsx}\"",
+    "build:docs": "lg-tsdoc",
     "chromatic": "npx chromatic",
     "clean": "yarn clean:builds && yarn clean:modules",
     "clean:builds": "rm -rf packages/*/dist packages/*/tsconfig.tsbuildinfo ./tsconfig.tsbuildinfo packages/**/stories.js",
     "clean:modules": "rm -rf node_modules packages/*/node_modules ",
-    "fix": "npm-run-all --parallel fix:*",
-    "fix:prettier": "prettier --write \"**/*.{js,ts,tsx,json,md,yml}\"",
-    "fix:eslint": "eslint \"**/*.{js,ts,tsx}\" --fix",
-    "link": "./scripts/link.sh",
-    "link:all": "lerna exec -- yarn link",
+    "fix": "lg-lint --fix",
+    "link": "lg-link",
+    "lint": "lg-lint",
     "prepublishOnly": "yarn fix && yarn test && yarn build && yarn prepublish:ts-downlevel",
     "prepublish:ts-downlevel": "rm -rf packages/*/dist/ts3.4 && echo packages/*/dist | NODE_NO_WARNINGS=1 xargs -I{} -n1 -P8 downlevel-dts {} {}/ts3.4",
     "publish": "npm-run-all --serial publish:*",
     "publish:packages": "yarn changeset publish --public",
```

Comment:
can we combine the two publish scripts into one?

Reply:
Maybe... I haven't dug into them, so I'm not 100% sure what they're doing